test/modules/test_nuclei.py

```diff
@@ -110,3 +111,26 @@ def test_group_targets_by_missing_tech(self) -> None:
 
         self.assertIn(frozenset(["wordpress"]), grouped_targets)
         self.assertEqual(grouped_targets[frozenset(["wordpress"])], expected_results[frozenset(["wordpress"])])
+
+    def test_lfi_dast_template(self) -> None:
+        task = Task(
+            {"type": TaskType.SERVICE.value, "service": Service.HTTP.value},
+            payload={
+                "host": "test-dast-vuln-app",
+                "port": 5000,
+            },
+        )
+        self.run_task(task)
+        (call,) = self.mock_db.save_task_result.call_args_list
+        self.assertEqual(call.kwargs["status"], TaskStatus.INTERESTING)
+        self.assertEqual(
+            call.kwargs["status_reason"],
+            "[high] http://test-dast-vuln-app:5000: Joomla! Helpdesk Pro plugin <1.4.0 - Local File Inclusion Directory traversal vulnerability in the Helpdesk Pro plugin before 1.4.0 for Joomla! allows remote attackers to read arbitrary files via a .. (dot dot) in the filename parameter in a ticket.download_attachment task., "
```

Comment:
can you mark this templte as suspicious in artemis/config.py? This is a standard flow if it trigges on a generic lfi